native/src/__mocks__/@react-native-community/netinfo.ts

```diff
@@ -30,23 +30,23 @@ export const addEventListener = (
 export const useNetInfo = (): NetInfoState => {
   throw Error('Not yet implemented in mock.')
 }
-export const removeEventListener = () => {
+export const removeEventListener = (): void => {
   throw Error('Function is deprecated')
 }
-export const getConnectionInfo = () => {
+export const getConnectionInfo = (): void => {
   throw Error('Function is deprecated')
 }
-export const isConnectionExpensive = () => {
+export const isConnectionExpensive = (): void => {
```

Comment:
`void` could be omitted because it is not really interesting I think.

Reply:
Not sure about this, it is still an information... hmh

Reply:
There is no easy solution to omit specific return types. (You can omit them for specific funtion names, arrow functions with const returns, ...)